packages/gammu/gammu.0.9.4/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler"
+           "Noémie Meunier"
+           "Pierre Hauweele"]
+tags: [ "SMS" "phone" "clib:gammu" ]
+license: "LGPL-3.0 with OCaml linking exception"
+homepage: "https://github.com/Chris00/ocaml-gammu"
+dev-repo: "https://github.com/Chris00/ocaml-gammu.git"
+bug-reports: "https://github.com/Chris00/ocaml-gammu/issues"
+doc: "https://Chris00.github.io/ocaml-gammu/doc"
+build: [
+  [ "jbuilder" "subst" ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "jbuilder"  {build & >="1.0+beta9"}
+  "configurator" {build}
```

Comment:
I checked your discover.ml file and saw that you're using stdio and base. I think that means you should list those as `{build}` dependencies as well.

Reply:
I agree — they are dependencies of `configurator` but adding them explicitly is safer.

Reply:
Thanks. It's best not to rely on transitive dependencies being available. Especially in this case, since we do want to get rid of extra dependencies in configurator.